### DISCUSSION TITLE
Does it support TCP BBR?

### DISCUSSION THREAD

**Discussion by symysak** [nextcloud/all-in-one#1296] [Questions] [answered]
Thanks for the great nextcloud.

Do you support the TCP BBR developed by Google?
I would like to enable this as I expect a significant speed increase when using it.

**Comment by szaimen** [comment, marked as answer]
Hi, I have the feeling that this is a Linux kernel setting: https://www.techrepublic.com/article/how-to-enable-tcp-bbr-to-improve-network-speed-on-linux/

**Reply by symysak** [reply]
Thank you for your reply.

If I make that setting in the container, won't it be reset when I restart the container?

**Reply by szaimen** [reply]
You need to apply the setting on your host and not in a container!

**Reply by symysak** [reply]
I see. My apologies.

Thank you very much.
